pkg/common/cns-lib/vsphere/virtualcenter.go

```diff
@@ -82,6 +82,9 @@ type VirtualCenterConfig struct {
 	Password string
 	// Insecure tells if an insecure connection is allowed.
 	Insecure bool
+	// Specifies the path to a CA certificate in PEM format. Optional; if not
+	// configured, the system's CA certificates will be used.
+	CAFile string
```

Comment:
`CAFile` specified here should be only used when `Insecure` flag is set to `false`.

What happens when `Insecure` is set to `true` and `CAFile` is specified in the config file?

Reply:
Thanks, I can update the logic to take that into account.

Just to make sure I understand correctly, does `Insecure` mean API calls are sent over plain HTTP, or that TLS is used with certificate validation disabled?

Reply:
@misterikkit 
insecure `true` means, Don't verify the server's certificate chain. same as curl's `-k` option - Allow connections to SSL sites without certs.
Refer this - https://github.com/vmware/govmomi/blob/master/vim25/soap/client.go#L146